r_scripts/data_wrangle/2013_oregon_case_crossover.R

```diff
@@ -0,0 +1,154 @@
+# ------------------------------------------------------------------------------
```

Comment:
rename this file to 2013-oregon_cardioresp_admissions. I only want this file to contain code that extracts the cvd and respiratory outcomes that are for the primary diagnosis. I want to keep only variables needed to create the time-stratified case cross over data frames latter. Variables to keep (dplyr select command): personkey, claimed, dx1, outcome, date, todate, fromdate, age, sex, and any other variable you can think of. This will differ slightly for pharmacy fills; probably ndc instead of dx1. I don't want variables never used in the analyses and that just clutter the data frame, making it hard to read a 'summary' or 'head'.